ethcore/types/src/transaction/error.rs

```diff
@@ -42,6 +42,14 @@ pub enum Error {
 		/// Transaction gas price
 		got: U256,
 	},
+	/// Transaction has too low fee
+	/// (there is already a transaction with the same sender-nonce but higher gas price)
```

Comment:
We need to document this better: what is the difference between `TooCheapToReplace` and `TxTooCheapToReplace` – it is not clear to when we return the one or the other.